curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.md

```diff
@@ -32,18 +32,18 @@ Next is the *anchor* element `<a>` (which requires a closing tag `</a>`):
 `target` is an anchor tag attribute that specifies where to open the link. The value `_blank` specifies to open the link in a new tab. The `href` is an anchor tag attribute that contains the URL address of the link:  
 
 ```html
-<a href="http://freecodecamp.org"> ... </a>
+<a href="http://freecodecamp.org" target="_blank"> ... </a>
```

Comment:
This is probably out-of-scope, but while we are at it, could you help update the URL as well? It's for consistency, and I personally feel a bit safer when I see a link with `https` protocol 😄 

```suggestion
<a href="https://www.freecodecamp.org" target="_blank"> ... </a>
```